### PR TITLE
Add Yandex.Translate

### LINKED ISSUE #2699: Yandex.Translate | translate.yandex.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
MaXfAa#1068

**What's the name of the service?** 
https://translate.yandex.com | Yandex.Translate

**What should the Presence display?** 
Same that Google Translate

**If possible, please provide a logo for the service (512x512 minimum)** 
https://s1.qwant.com/thumbr/0x380/b/8/159fcda2bd0b275590c209db0161034cf3178bd1c1e859f23cf0e82ec76144/Yandex_Translate_logo.jpg?u=http%3A%2F%2Fwww.velior.ru%2Fwp-content%2Fuploads%2F2018%2F10%2FYandex_Translate_logo.jpg&q=0&b=1&p=0&a=1

### PR DESCRIPTION
**Pull request by StrikerFRFX**
**Proof of Working:**
Discord_vxwcNZUs4l
Discord_JkoyZK7brv
Discord_zsLqvMD9MB
Discord_5qWTrX99ih
Discord_ak8W4Ez0C7

Closes #2699

### CODE REVIEW

**Comment by StrikerFRFX** [comment]
Oops left a redundant function in, will remove.

File: websites/Y/Yandex.Translate/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "url": [
+      "translate.yandex.com",
+      "translate.yandex.ru",
+      "ceviri.yandex.com.tr",
+      "translate.yandex.ua"
+    ],
+    "description": {
+      "en": "Yandex's free service instantly translates words, phrases, and web pages between English and 98 other languages!"
+    },
+    "service": "Yandex.Translate",
```
Comment (Bas950) [inline websites/Y/Yandex.Translate/dist/metadata.json:15]: Why with dot? Just use space? Same in folder name

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/Y/Yandex.Translate/dist/metadata.json (hunk 1 above)
Comment (StrikerFRFX) [reply websites/Y/Yandex.Translate/dist/metadata.json:15]: in the documentation for the website/the support page it is referenced to as Yandex.Translate, same with outside of the website (attached screenshots) 
Screenshot_20210115-170937_Chrome Beta
Screenshot_20210115-171010_Chrome Beta

Comment (StrikerFRFX) [reply websites/Y/Yandex.Translate/dist/metadata.json:15]: I was also keeping to the name of the original request (#2699)

Comment (Bas950) [reply websites/Y/Yandex.Translate/dist/metadata.json:15]: bruh

**Review by Bas950 — DISMISSED** [review]

File: websites/Y/Yandex.Translate/dist/metadata.json (hunk 1 above)
Comment (StrikerFRFX) [reply websites/Y/Yandex.Translate/dist/metadata.json:15]: yeah i know, consistency definitely isn't their thing it seems as Yandex Music has no dot. weird.

File: websites/Y/Yandex.Translate/presence.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        from = "Choosing...";
+        to = "Choosing...";
+      }
+    
+      const showTime: boolean = await presence.getSetting("stamp");
+    
+      presenceData.startTimestamp = showTime ? browsingStamp : null;
+      if (presenceData.startTimestamp == null) delete presenceData.startTimestamp;
+    
+      const showType: boolean = await presence.getSetting("type");
```
Comment (ririxidev) [inline websites/Y/Yandex.Translate/presence.ts:36]: just merge these consts 🤦

File: websites/Y/Yandex.Translate/presence.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    
+      if (showType) {
+        presenceData.details = `Translating: ${typet}`;
+        presenceData.state = `From: ${from} - To: ${to}`;
+      } else {
+        presenceData.details = `Translating from:  ${from}`;
```
Comment (ririxidev) [inline websites/Y/Yandex.Translate/presence.ts:42]: two spaces huh?

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/Y/Yandex.Translate/presence.ts (hunk 2 above)
Comment (StrikerFRFX) [reply websites/Y/Yandex.Translate/presence.ts:36]: Will merge sorry, forgot to go back and look over the code after I "finished" it

File: websites/Y/Yandex.Translate/presence.ts (hunk 3 above)
Comment (StrikerFRFX) [reply websites/Y/Yandex.Translate/presence.ts:42]: Typo

**Comment by StrikerFRFX** [comment]
ignore that re-request my browser lagged and it pressed it by accident

**Comment by StrikerFRFX** [comment]
> ignore that re-request my browser lagged and it pressed it by accident

unless I should've done that?

**Comment by StrikerFRFX** [comment]
> > ignore that re-request my browser lagged and it pressed it by accident
> 
> unless I should've done that?

nvm this it seems both reviews I requested merged so now I just look like an idiot rambling but before that it said requested from bas by itself and I assumed I wasn't supposed to do that

**Comment by StrikerFRFX** [comment]
closes #2699

**Review by Bas950 — APPROVED** [review]

**Comment by StrikerFRFX** [comment]
@ririxidev before you review! please review #2817 and read the NOTE at the bottom of that. Or at least read that. Due to my dumbass 2 day ago self PRing from master, if this is merged before #2817 then #2817 will remove all of these commits. @ me on discord if you want a full explanation of what I accidentally did 2 days ago, Striker#1337

**Comment by ririxidev** [comment]
!!!!!!!!!!!!!!!!!!!!!111111111

**Comment by StrikerFRFX** [comment]
> !!!!!!!!!!!!!!!!!!!!!111111111

I know, rookie mistake from 2 days ago with me having little github managing experience. Internally faceapalmed for the past few hours

**Pull request merged**